recipes/protobuf/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "4.23.1":
```

Comment:
out of curiosity.. what is the difference between 4.xx.y and 3.xx.y versions?
Looking at releases: they're marked as xx.y and tags 4.xx.y and 3.xx.y are equal

Reply:
Found above...
![image](https://github.com/conan-io/conan-center-index/assets/11200321/a28fcc1f-9fab-4f12-9bfa-bdf048895ce8)